src/components/select-field/props.js

```diff
@@ -3,6 +3,9 @@
 export type SelectOptionValueProp = string | number | null;
 
 export type SelectOptionProp = {
+    avatarUrl?: string,
```

Comment:
this is a little too specific to be used for all select options. which component is requiring this change?

Reply:
this is being used in the avatar

Reply:
I understand that, but an avatar URL is not part of the generic selector option when it comes to this field. and it doesnt easily map to any equivalent in a standard `<select>`. we should handle this in a way so that we do not have a sprawling list of one-off types for any current and future select options

Reply:
ok i'm goign to take this out for now because we are not doing images yet anyway